_posts/en/newsletters/2025-11-07-newsletter.md

```diff
@@ -0,0 +1,93 @@
+  compared over the decade.
+
+  The methodology was tested using 3 states that can be tested with functions in both libraries. A
+  benchmark was made with a list of pseudo-random key pairs. He ran the benchmark on 3 separate
+  machines and provided a bar plot. The chart showed that over the years, libsecp256k1 had improved
+  significantly while OpenSSL had remained the same.
+
+  Sebastion makes the conclusion that outside the Bitcoin ecosystem, the secp256k1 curve is not that
+  relevant and doesn't count as a first-class citizen, which justifies hours of work to improve. He
+  also encourages readers to try to reproduce his results and to report any issues in his
+  methodology or differences found in their results. The [source
+  code][libsecp benchmark code] is available on GitHub.
+
+## Changing consensus
+
+_A monthly section summarizing proposals and discussion about changing
+Bitcoin's consensus rules._
+
+FIXME:bitschmidty
+
+- **BIP54 implementation and test vectors**: Antoine Poinsot [posted][ap bip54
+  post] on the Bitcoin-Dev mailing list an update on his [Consensus
+  Cleanup][topic consensus cleanup] work, now [BIP54][] (see [Newsletter
+  #348][news348 bip54] for more details).
+
+  [PR99][binq bip54 pr] was opened against [Bitcoin Inquisition][bitcoin
+  inquisition repo] v29.1, implementing BIP54 consensus rules. This PR comes
+  with unit tests for each of the four mitigations, which can be used to
+  generate test vectors for the proposal. Moreover, it contains a fuzz harness
+  for the sigop accounting logic and functional tests that mimic mitigations
+  behaviour in realistic situations, including historical violations.
+  Additionally, a [custom miner][bip54 miner] was developed to generate a full
+  header chain needed by test vectors for mitigations requiring mainnet blocks,
+  such as timestamp and coinbase restrictions.
+
+  Finally, [PR2015][bips bip54-tv pr] in the [BIPs repository][bips repo] was
+  opened to add generated test vectors to BIP54.
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+FIXME:Gustavojfe
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+FIXME:Gustavojfe
+
+{% include snippets/recap-ad.md when="2025-11-11 16:30" %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="" %}
```

Comment:
```suggestion
{% include linkers/issues.md v=2 issues="2015" %}
```
@TumaBitcoiner for BIPs repo we can reference PRs numerically here and dont need the additional reference below